concepts/05 UI Components/Button/00 Getting started with Button/02 Create a Button.md

```diff
@@ -0,0 +1,121 @@
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#button").dxButton({
+            text: "Click me!"
+        });
+    });
+
+    <!-- tab: index.html -->
+    <html>
+        <head>
+            <!-- ... -->
+            <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.common.css">
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.light.css">
+            <link rel="stylesheet" href="index.css">
+
+            <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.all.js"></script>
+        </head>
+        <body class="dx-viewport">
+            <div id="button"></div>
+        </body>
+    </html>
+
+##### Angular
+
+[Add DevExtreme to your Angular application](/concepts/40%20Angular%20Components/10%20Getting%20Started/03%20Add%20DevExtreme%20to%20an%20Angular%20CLI%20Application '/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/') and use the following code to create a Button:
+
+    <!-- tab: app.component.html -->
+    <dx-button
+        text="Click me!">
+    </dx-button>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent { }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import {DxButtonModule} from 'devextreme-angular';
+    import { AppComponent } from './app.component';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxButtonModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+[Add DevExtreme to your Vue application](/concepts/55%20Vue%20Components/05%20Add%20DevExtreme%20to%20a%20Vue%20Application/00%20Add%20DevExtreme%20to%20a%20Vue%20Application.md '/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/') and use the following code to create a Button:
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxButton
+            text="Click me!"
+            @click="showMessage()"
+            styling-mode="outlined"
+            type="success"
+            icon="comment"
+        />
+    </template>
+
+    <script>
+    import DxButton from 'devextreme-vue/button';
+    import notify from 'devextreme/ui/notify';
+
+    export default {
+    name: 'App',
+    components: {
+        DxButton
+    },
+    methods: {
+        showMessage() {
+            notify("The button was clicked");
+        }
+    }
```

Comment:
These lines are misaligned, and the `showMessage` method shouldn't be illustrated yet.